repo2docker/buildpacks/conda/install-base-env.bash

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# This downloads and installs a pinned version of micromamba
+# and sets up the base environment
+set -ex
+
+
+cd $(dirname $0)
+
+URL="https://anaconda.org/conda-forge/micromamba/0.17.0/download/linux-64/micromamba-0.17.0-0.tar.bz2"
+
+# make sure we don't do anything funky with user's $HOME
+# since this is run as root
+unset HOME
+mkdir -p ${CONDA_DIR}
+
+time wget -qO- ${URL} | tar -xvj bin/micromamba
+chmod 0755 /tmp/bin/micromamba
+
+export MAMBA_ROOT_PREFIX=${CONDA_DIR}
+export MAMBA_EXE="/tmp/bin/micromamba"
+
+eval "$(/tmp/bin/micromamba shell hook -p ${CONDA_DIR} -s posix)"
+
+micromamba activate
+
+export PATH="${PWD}/bin:$PATH"
+
+cat <<EOT >> ${CONDA_DIR}/.condarc
+channels:
+  - conda-forge
+  - defaults
+auto_update_conda: false
+show_channel_urls: true
+update_dependencies: false
+# channel_priority: flexible
+EOT
+
+micromamba install conda mamba -y
```

Comment:
I think we should pin mamba and conda versions here, as well.

Reply:
Done.